### PR TITLE
Hyundai long improvements

### PR DESCRIPTION
**Pull request by haraschax**
Inspired by
https://github.com/commaai/openpilot/pull/24053

### CODE REVIEW

File: selfdrive/car/hyundai/hyundaican.py [hunk 1]
```diff
@@ -111,8 +111,8 @@ def create_acc_commands(packer, enabled, accel, jerk, idx, lead_visible, set_spe
   scc14_values = {
     "ComfortBandUpper": 0.0, # stock usually is 0 but sometimes uses higher values
     "ComfortBandLower": 0.0, # stock usually is 0 but sometimes uses higher values
-    "JerkUpperLimit": max(jerk, 1.0) if not stopping else 0, # stock usually is 1.0 but sometimes uses higher values
-    "JerkLowerLimit": max(-jerk, 1.0), # stock usually is 0.5 but sometimes uses higher values
+    "JerkUpperLimit": 10.0, # stock usually is 1.0 but sometimes uses higher values
+    "JerkLowerLimit": 10.0, # stock usually is 0.5 but sometimes uses higher values
```
[the remark this answers is not in the document]
Comment (sunnyhaibin) [reply selfdrive/car/hyundai/hyundaican.py:115]: Tried it this morning, it was undershooting/overshooting quite much in stop and go.

```36e10531feea61a4|2022-08-30--09-46-12``` (Please ignore the other alerts in the route 🙂)

**Comment by sunnyhaibin** [comment]
Tried it this morning, it was undershooting/overshooting quite much in stop and go.

```36e10531feea61a4|2022-08-30--09-46-12``` (Please ignore the other alerts in the route 🙂)

**Comment by sunnyhaibin** [comment]
Gave the new commits a go this morning, great improvements from yesterday!

`36e10531feea61a4|2022-08-31--09-37-30`
- The distance between the stopped lead car fluctuates between 2 to 7 meters, it's not very consistent, but does stop nicely and in time.
- Slowing down from a higher speed to stop was a bit abrupt in some instances
- Starting from standstill still takes a good bit after the lead car has taken off

**Comment by J4CY** [comment]
Tried out this branch this morning on my 21 Sonata Limited.
Route: `d015f0d097b9e23a|2022-08-31--08-11-16`

It seems that the car tends to stop way too close to the lead, I always end up taking over and braking as it's getting too close. It feels like same behavior as VOACC, although I have radar tracks enabled and have verified their presence in cabana. There are some occasional harsher braking but it's way better than the previous openpilot long on hyundais.

**Comment by gregjhogan** [comment]
Can we feed the jerk from the planner into the ESC jerk limits?  It seems like this should enable starting from a stop quickly without the harsh drop in accel upon transitioning back to the plan (seems like we need to feed large accel to the ESC until the brakes release to get going quickly, and an appropriate upper jerk limit to prevent the harsh drop transitioning back to the plan).

**Comment by HaraldSchafer** [comment]
Just inputting the planner jerk limits is probably not great, it kind of breaks the API boundary. It would bypass the PID and would ignore any acceleration error that could exist.

**Comment by gregjhogan** [comment]
> Just inputting the planner jerk limits is probably not great, it kind of breaks the API boundary. It would bypass the PID and would ignore any acceleration error that could exist.

You don't want to use the PID to get rolling from a stop.  Is there something better than the planner jerk to decide how fast to accelerate from a stop?  The sequence I use to get rolling (where I can control how fast to start rolling) ends before the car even starts moving, and the transition back to the plan is uncomfortable if you use too much jerk to get rolling.

**Pull request closed without merging**